README.md

```diff
@@ -57,6 +57,36 @@ Send your custom tags to Loggly by setting the `tag` property.
 
 ```Javascript
 _LTracker.push({
+  'logglyKey': '8c518f97-e3e0-4bfb-a8ed-582d084a5289',
+  'sendConsoleErrors' : true,
   'tag' : 'tag1,tag2'
 });
 ```
+
+Setup Proxy for Ad blockers
+----------
+You can proxy the requests from your own domain if the script or its requests are blocked by Ad blockers. To do this, you need to perform following steps
+
+Set `useProxyDomain` property to true
+
+```Javascript
+_LTracker.push({
+  'logglyKey': '8c518f97-e3e0-4bfb-a8ed-582d084a5289',
+  'sendConsoleErrors' : true,
+  'tag' : 'javascript-logs',
+  'useDomainProxy' : true
```

Comment:
Why not just use the domain proxy if it's not null? Wouldn't it be simpler to have one variable instead of two?

+_LTracker.push({
+  'logglyKey': '8c518f97-e3e0-4bfb-a8ed-582d084a5289',
+  'sendConsoleErrors' : true,
+  'tag' : 'javascript-logs',
+  'domainProxy' : 'loggly.mydomain.com'
+});